workflow-bot-app/src/test/java/com/symphony/bdk/workflow/utils/ExecuteRequestUtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.workflow.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.symphony.bdk.workflow.engine.executor.request.ExecuteRequestUtils;
+
+import org.junit.jupiter.api.Test;
+
```

Comment:
should we have a test for parameters that are already encoded?

Reply:
Added in next commit. We decided, in this case, to re-encode the query params without any check. If the workflow developer wants the url to be automatically encoded, he should not provide an encoded one. Otherwise, he can disable it using the optional field.